src/Diagnostics.AIProjects/TrainingFactory/metadata/trainingConfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "14748": {
+    "sof-tags": [
+      "azure-web-app-service",
+      "azure-webapps",
+      "azure-app-service",
+      "azure-web-app-for-containers",
```

Comment:
web app for containers in azure indicates linux containers. Will this create a problem?

Reply:
This is only a placeholder training config for now. In ideal situations we will be providing trainingConfig whenever we train. So, this is just a sample file for how training config should look like.